ui/components/Notification.js

```diff
@@ -0,0 +1,26 @@
+import { IconButton } from '@mui/material';
+import HighlightOffIcon from '@mui/icons-material/HighlightOff';
+
+export const closeButtonForSnackbarAction = (closeSnackbar) => (key) => (
+  <IconButton key="close" aria-label="Close" color="inherit" onClick={() => closeSnackbar(key)}>
+    <HighlightOffIcon />
+  </IconButton>
+)
+
+export const successHandlerGenerator = (snackbar, action, msg, cb) => (res) => {
+  if (res !== undefined) {
+    hideProgress()
```

Comment:
what is this `hideProgress` 😅  ?

Reply:
This is the progess bar which I will write but yeah I guess it will be good to remove now.